tests/expectations/derive_ostream.cpp

```diff
@@ -47,8 +47,62 @@ struct D {
   }
 };
 
+union F {
+  enum class Tag : uint8_t {
+    Foo,
+    Bar,
+    Baz,
+  };
+
+  friend std::ostream& operator<<(std::ostream& stream, const Tag& instance) {
+    switch (instance) {
+      case F::Tag::Foo: stream << "Foo"; break;
```

Comment:
Yeah, it seems a bit weird to use `Tag` in the declaration but use `F::Tag` here. Is this required? If so it'd be good to have a comment.

Reply:
Yeah the compiler was complaining about not knowing what `Tag` was if I just used `Tag::Foo` here. I was kinda surprised by it but didn't spend much time trying to figure it out. I can add a comment about it.